engine/manifest/manifest.ts

```diff
@@ -183,7 +189,7 @@ export const createResolver = <
 
   context.release = provider;
   const resolver = new ReleaseResolver<FreshContext>({
-    resolvers: { ...resolvers, ...defaultResolvers },
+    resolvers: { ...resolvers, ...defaultResolvers, ...freshResolvers },
```

Comment:
can you check if anywhere that is referencing defaultResolvers is also referencing freshResolvers now?

try to find for all "defaultResolvers" occurrence throughout the project and check if the fresh resolvers should be included there as well